snippets/visualbasic/VS_Snippets_CLR/string.intern/VB/string_intern.vb

```diff
@@ -1,25 +1,23 @@
-﻿'<snippet1>
-' Sample for String.Intern(String)
-Imports System.Text
+﻿Imports System.Text
 
 Class Sample
-   
-   Public Shared Sub Main()
-      Dim s1 As [String] = "MyTest"
-      Dim s2 As [String] = New StringBuilder().Append("My").Append("Test").ToString()
-      Dim s3 As [String] = [String].Intern(s2)
-      Console.WriteLine("s1 = '{0}'", s1)
-      Console.WriteLine("s2 = '{0}'", s2)
-      Console.WriteLine("s3 = '{0}'", s3)
-      Console.WriteLine("Is s2 the same reference as s1?: {0}", s2 Is s1)
-      Console.WriteLine("Is s3 the same reference as s1?: {0}", s3 Is s1)
-   End Sub 'Main
+
+    Public Shared Sub Main()
+        Dim s1 As [String] = "MyTest"
+        Dim s2 As [String] = New StringBuilder().Append("My").Append("Test").ToString()
+        Dim s3 As [String] = [String].Intern(s2)
+        Console.WriteLine($"s1 = {s1}")
+        Console.WriteLine($"s2 = {s2}")
+        Console.WriteLine($"s3 = {s3}")
+        Console.WriteLine($"Is s2 the same reference as s1?: {s2 Is s1}")
+        Console.WriteLine($"Is s3 the same reference as s1?: {s3 Is s1}")
+    End Sub 'Main
```

Comment:
```suggestion
    End Sub
```

Reply:
@rpetrusha , Actually, I've seen many examples that puts a comment beside `End Sub`s, `End Function`s and `End Class`s that mentions the Sub, Function, or class name.
If these should be removed, I think this could be broken down into small sub-directories in the repo and apply a regex to remove it in one sub-directory at time.

Reply:
That would be good, @Youssef1313. The End ' /<membername> and End ' \<classname> statements were written by porting tools and aren't helpful.